### PR TITLE
@kanaabe => Migrate to CircleCI

### PR DESCRIPTION
**Pull request by kanaabe**
Will likely need to be tweaked, but just to get the builds going to test this...

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  staging:
+    branch: master
+    owner: artsy
+    commands:
+      - NODE_ENV=production make deploy env=staging
+  production:
+    branch: release
+    owner: artsy
+    commands:
+      - NODE_ENV=production make deploy env=production
```
Comment (orta) [inline circle.yml]: This matches with https://github.com/artsy/metaphysics/blob/master/circle.yml - should be 👍

Comment (kanaabe) [inline circle.yml]: Oh wait but these won't actually push to their respective branches

Comment (orta) [inline circle.yml]: No, it'd mean your deploy process is `git push upstream master:staging` or `git push upstream master:release`

Comment (kanaabe) [inline circle.yml]: Right. Wonder which is better -- to use the CIRCLE_SHA or pull a sha with git? https://github.com/artsy/positron/blob/master/Makefile#L52

Comment (craigspaeth) [inline circle.yml]: You'll want to make sure to update the deploy command to take `$CIRCLE_SHA1` into consideration [here](https://github.com/artsy/positron/blob/master/Makefile#L53). e.g. `git push --force git@heroku.com:positron-$(env).git $CIRCLE_SHA1:refs/heads/master`.

Comment (orta) [inline circle.yml]: Clever

**Pull request merged**